### PR TITLE
Mission manager ptest case

### PR DESCRIPTION
**Pull request by tanishqaggarwal**
To run, invoke the command

`python -m ptest.run_simulation -c ptest/configs/fc_only_native.json -t DeploymentToStandby`

from within the psim/python directory. Be sure that you've activated the virtual environment and edited ptest/configs/fc_only_native.json to point to the correct location of FlightSoftware (or, if you're on Windows, use fc_only_teensy.json and then modify the USB port to which the Teensy is connected)

---
This change is [Reviewable](https://reviewable.io/reviews/pathfinder-for-autonomous-navigation/psim/128)

### CODE REVIEW

File: python/ptest/cases/base.py [hunk 1]
```diff
@@ -1,91 +1,200 @@
...
+
     @property
-    def run_sim(self):
-        return False
+    def sim_duration(self):
+        return 0
+
+    @property
+    def sim_initial_state(self):
+        return 'startup'
```
Comment (tanishqaggarwal) [inline python/ptest/cases/base.py:103]: We should try to wrap these properties up into a struct @nhz2 @kkrol27

File: MATLAB/sensor_reading.m [hunk 2]
```diff
@@ -70,6 +70,11 @@
     end
 end
 
+% This is a bandaid fix to let the DeploymentToStandby ptest case work.
+% The testcase depends on the attitude estimator working so that the satellite can
+% detumble quickly, but the estimator cannot work if the GPS position is NaN because
+% the GPS hasn't yet acquired lock. So this line of code forces GPS lock to exist.
+sensor_readings.position_ecef= position_ecef;
```
Comment (nhz2) [inline MATLAB/sensor_reading.m]: This should be in an if statement controlled by a variable in const, that way it won't break the matlab sim.

**Review by nhz2 — CHANGES_REQUESTED** [review]

**Review by nhz2 — APPROVED** [review]

**Review by kylekrol — APPROVED** [review]

**Comment by kylekrol** [comment]
@tanishqaggarwal What's the story here with this PR and the PTest merger into FSW?

**Comment by tanishqaggarwal** [comment]
Closing because the change has moved to FlightSoftware.

**Pull request closed without merging**